.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -145,57 +153,40 @@ cross_SubmitDataApp:
 cross_LambdaAPIGateway:
   title: Use &ABP; to invoke a &LAM; function
   title_abbrev: Use &ABP; to invoke a &LAM; function
-  synopsis: create an &LAMlong; function invoked by &ABPlong;.
+  synopsis: create an &LAMlong; function invoked by &ABPlong; that sends anniversary text messages.
   languages:
     Java:
       versions:
         - sdk_version: 2
           block_content: cross_LambdaAPIGateway_Java_block.xml
-          add_services:
-            dynamodb:
-            sns:
     JavaScript:
       versions:
         - sdk_version: 3
           block_content: cross_LambdaAPIGateway_JavaScript_block.xml
-          add_services:
```

Comment:
This file looks out of date because it's deleting some Python content. Can you make sure you merge in the latest?